<?xml version="1.0" ?>
<launch>

  <arg name="speed"	default="0.1"/>

  <!-- Start kitting client -->
  <node name="fetch_interactive"
	pkg="aist_routines" type="fetch_interactive.py" output="screen">
    <rosparam file="$(find aist_routines)/config/fetch_config.yaml"/>
    <param name="speed"		value="$(arg speed)"/>
  </node>

</launch>
